reference/faq.md

```diff
@@ -414,7 +414,7 @@ EOF
 
 {{site.prodname}} will then route traffic between {{site.prodname}} hosts using IP-in-IP.
 
-In AWS, you disable `Source/Dest. Check` instead of using IP-in-IP as long as all your instances are in the same subnet of your VPC.  This will provide the best performance.  You can disable this with the CLI, or right click the instance in the EC2 console, and select `Change Source/Dest. Check` from the `Networking` submenu.
+In AWS, you disable `Source/Dest. Check` instead of using IP-in-IP or VXLAN as long as all your instances are in the same subnet of your VPC. This will provide the best performance. You can [disable Source/Destination Check]({{ site.baseurl }}/reference/resources/felixconfig#spec) using [Felix configuration]({{ site.baseurl }}/reference/resources/felixconfig). Alternatively, you can also disable this with the AWS CLI, or right clicking each instances in the EC2 console, and select `Change Source/Dest. Check` from the `Networking` submenu.
```

Comment:
Suggestions for clarity. Generally, we should be general about UI/console steps as they can change.

For best performance in AWS, you can disable `Source/Dest. Check` instead of using IP-in-IP or VXLAN; but only if all your instances are in the same subnet of your VPC.  The setting must be `Disable` if you want the EC2 instance to process traffic not matching the host interface IP address. 

You can  [disable Source/Destination Check]({{ site.baseurl }}/reference/resources/felixconfig#spec) using [Felix configuration]({{ site.baseurl }}/reference/resources/felixconfig), the AWS CLI, or the EC2 console, Networking submenu where you can select `Change Source/Dest. Check` for each each instance.

Reply:
Thanks Barbara. This looks better.

I have used the suggested changes with some adjustments. Please see if it looks OK to you.